internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -872,3 +852,65 @@ func (r *AgentServiceConfigReconciler) newIngressCertCM(instance *aiv1beta1.Agen
 
 	return cm, mutateFn
 }
+
+func (r *AgentServiceConfigReconciler) newAssistedCM(instance *aiv1beta1.AgentServiceConfig, serviceURL *url.URL) (*corev1.ConfigMap, controllerutil.MutateFn) {
+	cm := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      serviceName,
+			Namespace: r.Namespace,
+		},
+	}
+
+	mutateFn := func() error {
+		if err := controllerutil.SetControllerReference(instance, cm, r.Scheme); err != nil {
+			return err
+		}
+
+		cm.Data = map[string]string{
+			"SERVICE_BASE_URL": getEnvVar("SERVICE_BASE_URL", serviceURL.String()),
```

Comment:
Can this env var hack go away now?

Reply:
@carbonin @djzager 
Not yet :laughing: 
https://github.com/openshift/assisted-service/pull/1778